lesson8.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+test_string = "Eat Work Play Sleep repeat"
+
+# Option 1: using index
+first_word_test_string = test_string[4:8]+"ing"
```

Comment:
Make sure you have spaces between `test_string[4:8]` and `"ing"`. This comment applies for line 13; line 14 is good!